common/fixation_entry_unique_index.go

```diff
@@ -0,0 +1,110 @@
+package common
+
+import (
+	"encoding/binary"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/common/types"
+)
+
```

Comment:
add some explanation in the beginning of the file

Reply:
done